src/native-stack/types.tsx

```diff
@@ -468,3 +473,53 @@ export type NativeStackDescriptor = Descriptor<
 export type NativeStackDescriptorMap = {
   [key: string]: NativeStackDescriptor;
 };
+
+// copy from Reanimated, to avoid strong dependency
+export type GoBackGesture =
+  | 'swipeRight'
+  | 'swipeLeft'
+  | 'swipeUp'
+  | 'swipeDown'
+  | 'verticalSwipe'
+  | 'horizontalSwipe'
+  | 'twoDimensionalSwipe';
+
+export type PanGestureHandlerEventPayload = {
```

Comment:
If you don't mind, I would like to stick with this name because it is the original name from GestureHandler. I suggest something like this:
<img width="514" alt="image" src="https://github.com/software-mansion/react-native-screens/assets/36106620/bc553859-ccd4-4c96-a879-c91736858234">
Is it ok?

Reply:
Yeah, sure!